test/check-machines-consoles

```diff
@@ -324,6 +325,104 @@ fullscreen=0
 
         self.waitViewerDownload("vnc", my_ip)
 
+    def testAddEditVNC(self):
+        b = self.browser
+
+        # Create a machine without any consoles
+
+        name = "subVmTest1"
+        self.createVm(name)
+
+        self.login_and_go("/machines")
+        self.waitPageInit()
+        self.waitVmRow(name)
+        self.goToVmPage(name)
+
+        # "Console" card shows empty state
+
+        b.wait_in_text(f"#vm-{name}-consoles .pf-v5-c-empty-state", "Graphical support not enabled.")
+        b.assert_pixels(f"#vm-{name}-consoles", "no-vnc")
+
+        b.click(f"#vm-{name}-consoles .pf-v5-c-empty-state button:contains(Add VNC)")
+
+        b.wait_visible("#add-vnc-dialog")
+        b.set_checked("#add-vnc-autoport", val=True)
+        b.set_input_text("#add-vnc-port", "5000")
+        b.click("#add-vnc-add")
```

Comment:
While this is type=text (see question above), this should also test a non-numeric value.

Reply:
Yes, done.